src/modules/helpers.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable no-console */
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/prefer-default-export */
```

Comment:
- [ ] KIndly remove this & solve the linter manually, if the error present.

Reply:
well resolved! ✔️